lib/slack/agent/generateAgentResponse.ts

```
@@ -246,7 +263,7 @@ export const generateAgentResponse = async (
           sentBy:
             message.role === "user"
               ? message.emailFrom
-              : getFullName(members.find((member) => member.id === message.userId)!),
+              : getFullName(members.find((member) => member.id === message.userId)!.displayName, members.find((member) => member.id === message.userId)!.email),
```

ESLint

[error] Replace `members.find((member)·=>·member.id·===·message.userId)!.displayName,·members.find((member)·=>·member.id·===·message.userId)!.email` with `⏎··················members.find((member)·=>·member.id·===·message.userId)!.displayName,⏎··················members.find((member)·=>·member.id·===·message.userId)!.email,⏎················`

(prettier/prettier)